ext/standaloneusers/sql/auto_uninstall.sql

```diff
@@ -15,7 +15,7 @@
 
 SET FOREIGN_KEY_CHECKS=0;
 
-DROP TABLE IF EXISTS `civicrm_user`;
+DROP TABLE IF EXISTS `civicrm_uf_match`;
```

Comment:
This is slightly alarming... normally it shouldn't ever run but... 
I would delete this line but it's an autogenerated file. Wish there was a way to suppress a drop table in codeGen.